--- libcdio-paranoia-10.2+2.0.2/include/cdio/paranoia/paranoia.h	2024-07-20 14:16:30.000000000 +0200
+++ libcdio-paranoia-10.2+2.0.2/include/cdio/paranoia/paranoia.h	2026-08-13 23:25:47.458823204 +0200
@@ -28,7 +28,7 @@
 #ifndef CDIO__PARANOIA__PARANOIA_H_
 #define CDIO__PARANOIA__PARANOIA_H_
 
-#include <cdio/paranoia/cdda.h>
+#include <cdio/cdda.h>
 
 /*! Paranoia likes to work with 16-bit numbers rather than
     (possibly byte-swapped) bytes. So there are this many
